### PR TITLE
Checkered background

### PR DESCRIPTION
**Pull request by Reettaphant**
Checkered background for transparent images:
screen shot 2016-05-26 at 14 51 09

### CODE REVIEW

**Comment by paperboyo** [comment]
Wooo - stylish! Is it on a search page (in browser) or on an image page (in viewer)? Or on both?

Often to judge quality of a cutout, it's good practice to check it against white *and* against black. If it works there - will likely work everywhere (on all our card tones).

Checkered background is useful to distinguish between cutouts and non-cutouts quickly on a search page. Not so much to judge the cutout itself.

Just saying, because I'm not sure I would suddenly want to have a background colour switcher...

**Comment by Reettaphant** [comment]
Do we want to have a checkered background behind transparent images? 

@blishen @NickPapacostas

**Comment by paperboyo** [comment]
Not that we should emulate anyone... Here is Photoshop:

picture 21

How hard would it be to have three options:
- Grid's dark background
- something whiteish
- checkered

Easiest would be if one could cycle between them by e.g. Shift+clicking on an image and the choice would be remembered in localstorage.

All of the above, because I can't make up my mind what would be universally most useful... Ignore at will.

**Comment by NickPapacostas** [comment]
@Reettaphant re: checkered background, i think it's a good idea. But couldn't we just do it really simply for V1 (i.e. it just defaults to checkered )

**Comment by Reettaphant** [comment]
@NickPapacostas @paperboyo I agree with Nick. If it looks like something useful as it is, I'd like to release it now and then, if there is a need, perhaps look into making the UI more complex later.

**Comment by paperboyo** [comment]
I'm extremely malleable. I agree with the majority. Also: always great to have a reason for moaning later 😜

Maybe just something a tad lighter than black checkers? Hard to see details of a cutout otherwise:
picture 22

File: kahuna/public/js/image/view.html [hunk 1]
```diff
@@ -190,7 +192,7 @@
                ui:drag-data="ctrl.image | asImageAndCropsDragData:ctrl.crop"
                ui:drag-image="extremeAssets.smallest | assetFile">
                 <!-- TODO: Add tracking to crop -->
-                <img class="easel__image"
+                <img class="easel__image checkered__background"
```
Comment (NickPapacostas) [inline kahuna/public/js/image/view.html]: As I've recently learned thanks to Shaun and Akash, we're trying to use [BEM](https://css-tricks.com/bem-101/) for selectors. 

so i _think_ this becomes 

``` HTML
<img class="easel__image easel__image--checkered__background"
```

**Comment by Reettaphant** [comment]
@paperboyo What do you think of the new colouring scheme?

**Comment by paperboyo** [comment]
@Reettaphant Perfect!

**Comment by akash1810** [comment]
👍 :shipit:

**Pull request merged**